fresque/lib/alembic/env.py

```diff
@@ -1,17 +1,26 @@
-from __future__ import with_statement, unicode_literals, absolute_import, print_function
```

Comment:
@abompard I was trying to run the server for the first time and I was getting this error https://gist.github.com/rahulrrixe/a4d2ee2627210320d07c as alembic versions folder was not available. So I ran the alembic init for initialisation.